### ISSUE TITLE
Staging Review finding: Image is removed at high zoom

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#86833]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** VYE Team
**Product:** Verify Your Enrollment
**Feature:** Verify Your Enrollment

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/mgib-enrollments/benefits-profile/](https://staging.va.gov/education/verify-school-enrollment/mgib-enrollments/benefits-profile/) at high zoom when changing direct deposit information, the image of a personal check is removed from the document at high zoom (300% and higher).

This impacts two big groups of users:
- Low-vision users who use browser zoom, who will no longer be able to see the illustration of the check.
- Screen reader users on mobile devices, who will no longer hear the alt text indicating where to find the routing and account numbers. (Note that many blind Veterans, especially younger ones, use their phones as their primary --- and sometimes only --- device.)

## Recommended action

All content on the page needs to be available at all zoom levels up to 400% (equivalent to 320px width).

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 1.4.10 Reflow - Level AA](https://www.w3.org/TR/WCAG21/#reflow)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jsimonVA** [comment]
Image shows in mobile view and 400%+ zoom
https://github.com/department-of-veterans-affairs/vets-website/pull/30528

**Comment by wafimohamed** [comment]
removed radio button when there is no suggested address 
https://github.com/department-of-veterans-affairs/vets-website/pull/30537

**Comment by jsimonVA** [comment]
Update has been made and is in staging.
